File: Server 1.java
import java.net.ServerSocket;
import java.net.Socket;
import java.io.DataInputStream;
import java.io.DataOutputStream;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Server {
    public static void main(String[] args) {
        try {
            ServerSocket ss= new ServerSocket(5004);
            System.out.println("Server has been created, waiting for client");
            Socket s = ss.accept();
            System.out.println("Client has been connected");
            DataInputStream din = new DataInputStream(s.getInputStream());
            DataOutputStream dout = new DataOutputStream(s.getOutputStream());
            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            String str = " ", str2 = " ";
            while (!str.equals("stop")) {
                str = (String)din.readUTF();
                System.out.println("Client: "+str);
                str2 = br.readLine();
                dout.writeUTF(str2);
                dout.flush();
            }
            din.close();
            dout.close();
            ss.close();
        } catch (Exception e) {
            System.out.println(e);
        }
    }
}

File: server1.py
import socket

HOST = 'localhost'
PORT = 5005

server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
server.bind((HOST, PORT))
server.listen(1)

print("Server has been created, waiting for client...")

conn, addr = server.accept()
print("Client connected:", addr)

while True:
    data = conn.recv(1024).decode()
    if not data:
        break

    print("Client:", data)

    if data.lower() == "stop":
        break

    reply = input("You: ")
    conn.send(reply.encode())

conn.close()
server.close()